ui/src/CollectionExerciseDetails.js

```diff
@@ -568,6 +568,20 @@ class CollectionExerciseDetails extends Component {
             </div>
           </DialogContent>
         </Dialog>
+        {(this.state.authorisedActivities.includes("LOAD_BULK_REFUSAL") ||
+          this.state.authorisedActivities.includes(
+            "VIEW_BULK_REFUSAL_PROGRESS"
```

Comment:
this whole statement can be written a lot more compact by doing this:

`["LOAD_BULK_REFUSAL", "VIEW_BULK_REFUSAL_PROGRESS].includes(this.state.authorisedActivities)`